src/node_main.cc

```diff
@@ -88,42 +88,8 @@ int wmain(int argc, wchar_t* wargv[]) {
 }
 #else
 // UNIX
-#ifdef __linux__
-#include <sys/auxv.h>
-#endif  // __linux__
-#if defined(__POSIX__) && defined(NODE_SHARED_MODE)
-#include <string.h>
-#include <signal.h>
-#endif
-
-namespace node {
-namespace per_process {
-extern bool linux_at_secure;
-}  // namespace per_process
-}  // namespace node
 
 int main(int argc, char* argv[]) {
-#if defined(__POSIX__) && defined(NODE_SHARED_MODE)
-  // In node::PlatformInit(), we squash all signal handlers for non-shared lib
-  // build. In order to run test cases against shared lib build, we also need
-  // to do the same thing for shared lib build here, but only for SIGPIPE for
-  // now. If node::PlatformInit() is moved to here, then this section could be
-  // removed.
```

Comment:
This change does the reverse, move this code to `PlatformInit()`, but the result is the same.